charts/projectsveltos/templates/deployment.yaml

```diff
@@ -435,14 +492,31 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       containers:
-{{- if .Values.agent.managementCluster }}      
-      - args: {{- toYaml .Values.classifierManager.manager.argsAgentMgmtCluster | nindent 8 -}}
-{{ else }}
-      - args: {{- toYaml .Values.classifierManager.manager.args | nindent 8 -}}
-{{- end }}
-{{- if ne (len .Values.classifierManager.agentPatchConfigMap.data) 0 }}
-        - --sveltos-agent-config={{ .Values.classifierManager.agentPatchConfigMap.name }}
-{{- end }}
+    {{- if .Values.agent.managementCluster }}
+      - args:
+      {{- range .Values.classifierManager.manager.argsAgentMgmtCluster }}
+        - {{ . }}
+      {{- end }}
+      {{- range $key, $value := .Values.classifierManager.manager.extraArgsAgentMgmtCluster }}
```

Comment:
Same comment as above. Maybe I am reading it incorrectly, but I believe we are removing Values.classifierManager.agentPatchConfigMap